### PR TITLE
chore(deps): drop html-tags

### PR DESCRIPTION
**Pull request by Tahul**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [X] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [X] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Drop `html-tags` dependency as it was breaking builds in some contexts.

As it is a very simple array of HTML tags, I think we can inline it as we do for PROSE_TAGS.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "test:coverage": "vitest --coverage",
     "test:types": "tsc --build tsconfig.json",
     "test": "./scripts/test.sh",
+    "prepare": "nuxi prepare playground/basic",
```
Comment (farnabaz) [inline package.json:43]:
```suggestion
```

We have `dev:prepare` in scripts

File: src/runtime/utils/html-tags.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
+export default [
```
Comment (farnabaz) [inline src/runtime/utils/html-tags.ts:1]:
```suggestion
// Ref https://github.com/sindresorhus/html-tags/blob/v3.2.0/html-tags.json
export default [
```

Do you mind also fixing lints?

Comment (Tahul) [reply src/runtime/utils/html-tags.ts:1]: Sure

File: package.json (hunk 1 above)
Comment (Tahul) [reply package.json]: Moved `dev:prepare` into `prepare` so it runs at the end of install script :)

**Review by farnabaz — APPROVED** [review]

**Pull request merged**